src/main/kotlin/calculator/interfaces/StringReadStrategy.kt

```diff
@@ -0,0 +1,5 @@
+package calculator.interfaces
```

Comment:
비즈니스 문제를 해결하는 클래스들은 `domain` 패키지에 위치하면 어떨까요?

```suggestion
package calculator.domain
```